sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-backend-v2/README.md

```diff
@@ -80,10 +80,10 @@ spring.security.oauth2.client.registration.azure.redirect-uri-template={baseUrl}
 spring.security.oauth2.client.registration.azure.client-authentication-method=basic
```

Comment:
I think there is an error here, `spring.security.oauth2.client.registration.azure.redirect-uri-template` should update to `spring.security.oauth2.client.registration.azure.redirect-uri`
and do we need to add `profile` in `spring.security.oauth2.client.registration.azure.scope`?

Reply:
yes, the profile needs to be updated too.